features/FEATURE_BLE/targets/TARGET_CORDIO_LL/stack/controller/sources/common/chci/chci_tr.c

```diff
@@ -401,6 +401,8 @@ static void chciTrWrite(uint8_t prot, uint8_t type, uint16_t len, uint8_t *pData
   PalSysSetBusy();
 #else
   FakeChciTrWrite(prot, type, len, pData);
+  chciTrCb.wrBufComp = TRUE;
```

Comment:
what's the impact of servicing immediately ? Why are we not checking the return of `FakeChciTrWrite` ?

Reply:
We service immediately because there is nothing else to run ChciTrService() and what is the point in waiting? We are the only ones producing writes.

The is no point checking the return since there is no way to handle error. The packet is just lost. There are no error codes in cordio - it assumes every function always works.

Reply:
Thanks for the clarification about servicing immediately  the request. Could you check with the Cordio team if this change can be upstreamed ? 

> The is no point checking the return since there is no way to handle error.
I think there is a point, we should release the memory but I might be wrong.

Reply:
There is nothing to release, the data is in a global tx buffer (failed packets are in effect dropped).

Reply:
I'll make a jira ticket to  upstream the service call